astropy/stats/tests/test_circstats.py

```diff
@@ -0,0 +1,250 @@
+    # Ref[1], page 23
+    # testing non-Quantity input
+    data_deg = np.array([51, 67, 40, 109, 31, 358])
+    data_rad = np.deg2rad(data_deg)
+    # 2nd, 3rd, and 4th moments
+    answer_rad = np.array([1.588121, 1.963919, 2.685556])
+    # testing input in radians
+    data_rad = data_rad*u.rad
+    angles_answer_rad = answer_rad*u.rad
+    angles_result = (np.around(circmoment(data_rad, p=2)[0], 6),
+                     np.around(circmoment(data_rad, p=3)[0], 6),
+                     np.around(circmoment(data_rad, p=4)[0], 6))
+    assert_equal(angles_answer_rad[0], angles_result[0])
+    assert_equal(angles_answer_rad[1], angles_result[1])
+    assert_equal(angles_answer_rad[2], angles_result[2])
+    # testing input in degrees
+    data_deg = np.rad2deg(data_rad)
+    angles_answer_deg = np.around(np.rad2deg(angles_answer_rad), 4)
+    angles_result = (np.around(circmoment(data_deg, deg=True, p=2)[0], 4),
+                     np.around(circmoment(data_deg, deg=True, p=3)[0], 4),
+                     np.around(circmoment(data_deg, deg=True, p=4)[0], 4))
+    assert_equal(angles_answer_deg[0], angles_result[0])
+    assert_equal(angles_answer_deg[1], angles_result[1])
+    assert_equal(angles_answer_deg[2], angles_result[2])
+    # testing lengths
+    answer = np.array([0.4800428, 0.236541, 0.2255761])
+    assert_allclose(answer, (circmoment(data_rad, p=2)[1],
+                             circmoment(data_rad, p=3)[1],
+                             circmoment(data_rad, p=4)[1]), atol=1e-4)
+    assert_allclose(answer, (circmoment(data_deg, deg=True, p=2)[1],
+                             circmoment(data_deg, deg=True, p=3)[1],
+                             circmoment(data_deg, deg=True, p=4)[1]),
+                    atol=1e-4)
+
+
+def test_circcorrcoef():
+    # testing against R CircStats package
+    # Ref[1], page 180
+    # testing non-Quantity
+    alpha_deg = np.array([356, 97, 211, 232, 343, 292, 157, 302, 335, 302, 324,
+                          85, 324, 340, 157, 238, 254, 146, 232, 122, 329])
+    beta_deg = np.array([119, 162, 221, 259, 270, 29, 97, 292, 40, 313, 94, 45,
+                         47, 108, 221, 270, 119, 248, 270, 45, 23])
+    alpha_rad = np.deg2rad(alpha_deg)
+    beta_rad = np.deg2rad(beta_deg)
+    answer = 0.2704648
+    # testing input in degrees
+    assert_allclose(answer, circcorrcoef(alpha_deg, beta_deg, deg=True),
+                    atol=1e-4)
+    # testing input in radians
+    assert_allclose(answer, circcorrcoef(alpha_rad, beta_rad), atol=1e-4)
+    # testing with Quantity
+    # testing input in degrees
+    alpha_deg = alpha_deg*u.deg
+    beta_deg = beta_deg*u.deg
+    assert_allclose(answer, circcorrcoef(alpha_deg, beta_deg, deg=True),
+                    atol=1e-4)
+    # testing input in radians
+    alpha_rad = alpha_rad*u.rad
+    beta_rad = beta_rad*u.rad
+    assert_allclose(answer, circcorrcoef(alpha_rad, beta_rad), atol=1e-4)
+
+
+def test_rayleightest1():
```

Comment:
Can you give this a more descriptive name than just test1?   Or at least a comment about what is different between test1 and test2 and what they are suppose to specifcially  test?